_data/production_units.yml

```diff
@@ -2,14 +2,15 @@
 mcf:
   long: mcf
   short: mcf
+  term: mcf
 tons:
   long: tons
   short: tons
   term: ton
 short tons:
-  long: short tons
-  short: short tons
-  term: short ton
```

Comment:
This was not pulling in the glossary term to tons because `short ton` is not a term in the glossary